homeassistant/components/media_player/__init__.py

```diff
@@ -861,6 +887,14 @@ async def async_browse_media(
         """
         raise NotImplementedError()
 
+    async def async_join_players(self, group_members):
```

Comment:
We normally implement sync versions too of the service handlers and have these async coroutine functions await the sync methods scheduled in the executor with `hass.async_add_executor_job`. Raise NotImplementedError in the sync methods.

Reply:
@MartinHjelmare Sorry about the squashing. I have now added a sync version of the two methods as well.